PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -65,6 +65,8 @@ class PsfPowershellScriptRunner
     }
 
 private:
+    DWORD createInContainerAttribute = 0x02;

```

Comment:
Add comment here that this is really the PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_DISABLE_PROCESS_TREE value.  It's public and documented here,
    [https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute](https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute)

Also mention in comment that child and descendant processes of powershell will be bound to the container when this attribute is set.